README.md

```diff
@@ -71,7 +72,7 @@ git clone https://github.com/faisalakb/Ruby_Morse_Code.git
 Install this project with:
 
 ```
-cd cd Create-your-own-Enumerable
+cd Ruby_Morse_Code
 gem install
 ```
 
```

Comment:
Great job team 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.
![Issues page](https://github.com/faisalakb/Ruby_Morse_Code/assets/63917725/739bfc0d-1a73-4e9c-8cbc-6be311a51d36)

Reply:
- [ ] Good job in implementing the changes requested by the previous reviewer, :clap:  however as he mentioned above please ensure that you solve your broken issues link. Having broken links in your documentation can make your project or organization appear unprofessional and unreliable. Users may question the accuracy and trustworthiness of your content. :slightly_smiling_face: 

Below is the screenshot: :arrow_down: 
![Issu](https://github.com/faisalakb/Ruby_Morse_Code/assets/43528594/a9324ad2-b065-4a4b-8dd1-390c64bcfe88)